### DISCUSSION TITLE
Please clarify License requirments

### DISCUSSION THREAD

**Discussion by dbgnvan2** [brianpetro/obsidian-smart-connections#683] [Q&A]
I'm have a very hard time getting SC - Obsidian to do anything.  Please deny, confirm, or clarify the following.  I'm not a programmer. 
1. Do I have to have SC Support license get functionality in Obsidian. 
2. I MUST have a paid subscription to use OpenAI - Api Key with SC in Obsidian 
3. I have a Open AI plus license - do I have to upgrade / change API Keys SC in Obsidian to work. 
4. Is there an Out of the Box default set up for OBSIDIAN that will work with my notes?
5. Is there basic tutorial on Youtube to get this to work ( happy path )?

I'm trying to using SC.  I've been using Obsidian for over a year.   thanks

**Comment by brianpetro** [comment]
Hi @dbgnvan2 

1. No. All functionality works _without_ a supporter key.

2. If you want to use OpenAI then you should become familiar with their product. Neither OpenAI API account or ChatGPT subscription is required to use Smart Connections. See 4 for more.

3. No, see 2 and 4 for more.

4. You do not need to use OpenAI. By default, Smart Connections uses a local embedding model, powering the Smart View. And the Smart Chat defaults to using Open Router which has many options for running cloud models, including free options. Other cloud model platforms are available, and so is configuring local chat models.

5. I need to make more videos that focus on basic usage, but the plugin is designed to work out of box, immediately after you install it, including limited access to a free chat model, so most the how-tos will probably need to revolve around workflows rather than setup. This playlist has some videos https://youtube.com/playlist?list=PLqu2iXqr-g4wXYDEXTLgoEP3g5vS73hwj&si=Wt6RpRL8sGNLGyyv

🇺🇸🌴

**Reply by dbgnvan2** [reply]
Wow- that was quick.  I'll keep plugging away.
